python/gubbins/common.py

```diff
@@ -864,6 +874,18 @@ def transfer_internal_node_labels_to_tree(source_tree_filename, destination_tree
                                         suppress_internal_node_labels=True).replace("'","").replace('\'', ''),
                 file=output_file)
 
+def remove_internal_nodes_from_alignment(input_filename):
+    # Get the start line of the internal nodes in the alignment
+    start_cmd = "grep -n '^>internal_Node1$' " + input_filename + " | sed 's/\:.*$//g' "
+    start_num = subprocess.check_output(start_cmd, shell=True)
+    start_num = str(int(start_num.strip().decode("utf-8")) - 1)
+
+    # Now just use head to take all the lines before this
+    head_cmd = "head -n " + start_num + " " + input_filename + " > tmp_aln.txt && mv tmp_aln.txt " + input_filename
+    try:
+        subprocess.check_call(head_cmd, shell=True)
+    except subprocess.SubprocessError:
+        sys.exit("Command to remove internal nodes from raxmlng ancestral alignment failed")
 
```

Comment:
Debugging statements?

Reply:
Old function that wasn't needed upon further investigation, removed now